code/modules/shuttle/gamemodes/crash.dm

```diff
@@ -58,7 +58,7 @@
 */
 // -- Shuttles
 
-/obj/docking_port/stationary/crashmode/bigbury
+/obj/docking_port/stationary/crashmode/bigbury // How-to-use: Put ontop/below of an allready existing crash dock site.
```

Comment:
don't need this just increase the size of the standard crash docking port.

Reply:
As you wish, my lord.